frameworklauncher/src/test/resources/TestExpects/PlatformSpecificParametersDescriptor.yml

```diff
@@ -24,6 +24,7 @@ generateContainerIpList: false
 killAllOnAnyCompleted: false
 killAllOnAnyServiceCompleted: false
 queue: testString
+skipLocalTriedResource: false
```

Comment:
>kipLocalTriedResource: fals [](start = 1, length = 27)

the default value need to be true.

Reply:
It is just UT yaml

---
In reply to: [193963799](https://github.com/Microsoft/pai/pull/685#discussion_r193963799) [](ancestors = 193963799)